releasenotes/sfdx/README.md

```diff
@@ -14,7 +14,20 @@ If you use [autocomplete](https://developer.salesforce.com/docs/atlas.en-us.sfdx
 
 Want to check out the new `sf` executable of Salesforce CLI? [Click here for the release notes.](../sf/README.md)
 
-## 7.144.2 (March 31, 2022) [stable-rc]
+## 7.145.0 (April 7, 2022) [stable-rc]
+
+These changes are in the Salesforce CLI release candidate. We plan to include these changes in next week's official release. This list isn't final and is subject to change.
+
+* FIX: Setting your default Dev Hub while authorizing it now immediately propagates to your whole CLI environment. As a result, valid scratch org deletions no longer occasionally fail with error `Unable to associate this scratch org with a DevHub`. (GitHub issue [#1423](https://github.com/forcedotcom/cli/issues/1423))
```

Comment:
I'm not totally clear how this works in the case where you have multiple Dev Hubs.

Reply:
"propagate to your whole CLI environment" is just a way of saying we now do it correctly, rather than before where internal config files in .sfdx didn't get a specific property updated until you run another command.  As a result, deleting scratch orgs associated with that dev hub would sometimes fail, incorrectly.  But I don't want to talk about internal configuration files (we don't want people to mess with them) so I used that propagating phrase.  Lemme know if you think of a better one!

Reply:
Can't think of anything better